### DISCUSSION TITLE
Unable to set google drive as a backup disk

### DISCUSSION THREAD

**Discussion by Naveen-474** [spatie/laravel-backup#1764] [Q&A]
While using local has a backup disk, my Laravel project works fine!!
When using google Drive, it throws an error like

Copying zip failed because: Could not connect to disk google because: League\Flysystem\UnableToReadFile: Unable to read file from location: sm-uniforms-development. File not found in /var/www/html/sm-uniforms-development/vendor/league/flysystem/src/UnableToReadFile.php:24

Note: Google Drive works fine for another job that simply generates PDFs and stores them on the drive.

**Comment by fredsal** [comment]
It is working for me

**Comment by lowv-developer** [comment]
I have the same problem
